.laminas-ci.json

```diff
@@ -1,5 +1,5 @@
 {
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.2": true
```

Comment:
Probably not needed anymore? 🤔

Reply:
laminas/laminas-captcha has php8.2 support as this required php8.2 support

Reply:
Yes, but are current dependencies precluding a PHP 8 upgrade? This file introduces a CI error suppression/workaround, so it should be possible to remove it, if all dependencies are 8.2-ready

Reply:
Yes all dependencies have 8.2 support

Reply:
Then this file can be dropped 👍

Reply:
👍